document.go

```diff
@@ -95,6 +94,26 @@ func (doc *Document) Create(data []byte) (err error) {
 	return
 }
 
+func (doc *Document) Delete() (err error) {
+	if doc.Id == "" {
+		return errors.New("An id required to delete a document.")
+	}
+	_, err = doc.getDocFromId()
+	if err != nil {
+		return err
+	}
+	_, body, errs := doc.Db.Req.Delete(doc.Id).Query("rev=" + doc.Rev).End()
+	log.Debug("Deleting " + doc.Id)
+	log.Debug("Delete Rev " + doc.Rev)
+	log.Debug("Delete Body " + body)
+	if len(errs) != 0 {
```

Comment:
Can you just combine these errors into one and `return err` instead of having `(err error)` as the function signature?